### PR TITLE
Update dev series readme

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,14 @@ HTCondor-CE RPMs are available from the following locations:
 
 - HTCondor Yum repositories: https://research.cs.wisc.edu/htcondor/yum/
 - OSG Yum repositories: https://opensciencegrid.org/docs/common/yum/
+
+Versioning
+----------
+
+HTCondor-CE follows the same stable/development release series model as [HTCondor](https://htcondor.readthedocs.io/en/latest/version-history/introduction-version-history.html).
```
Comment (matyasselmeci) [inline README.md]:
```suggestion
HTCondor-CE follows the same even stable/odd development release series model as [HTCondor](https://htcondor.readthedocs.io/en/latest/version-history/introduction-version-history.html).
```
That way most people won't have to follow the link to understand the sentence.

Comment (brianhlin) [reply README.md]: That isn't the case, though? 3 is stable, 4 is devel

Comment (matyasselmeci) [reply README.md]: So it's the opposite of the Condor versioning scheme (which explicitly says "Even numbers represent stable versions, while odd numbers represent development versions.").
Maybe just drop that sentence and rephrase the paragraph as follows:
```
As of September 2019, we maintain two different versions of HTCondor-CE:

- HTCondor-CE 4 ([documentation](https://htcondor-ce.readthedocs.io/en/latest/)),
  kept in the [master](https://github.com/htcondor/htcondor-ce/tree/master) branch
- HTCondor-CE 3 ([documentation](https://htcondor-ce.readthedocs.io/en/stable/)),
  kept in the [stable](https://github.com/htcondor/htcondor-ce/tree/stable) branch
```

File: README.md [hunk 2]
```diff
@@ -24,3 +24,14 @@ HTCondor-CE RPMs are available from the following locations:
...
 - HTCondor Yum repositories: https://research.cs.wisc.edu/htcondor/yum/
 - OSG Yum repositories: https://opensciencegrid.org/docs/common/yum/
+
+Versioning
+----------
+
+At any given time, two versions of HTCondor-CE are maintained, a stable and a development version.
+In this repository, the `master` branch contains the latest version of HTCondor-CE (i.e. development) while the `stable`
+branch contains the previous version.
+As of September 2019, we maintain two different versions of HTCondor-CE:
```
Comment (matyasselmeci) [inline README.md]:
```suggestion
```
With your new sentence, this is a dupe.

Comment (matyasselmeci) [reply README.md]: Alternatively, phrase it as
"As of September 2019, the two different versions of HTCondor-CE we maintain are:"

**Comment by matyasselmeci** [comment]
Works for me. Feel free to merge once the CI passes.

**Comment by brianhlin** [comment]
CI has actually passed but it looks like Travis is having some issues so it's not reflected in GitHub. Merging.

**Pull request merged**